src/containers/Storage/utils/constants.ts

```diff
@@ -1,10 +1,3 @@
-export const colorSeverity = {
```

Comment:
this was actually a reverse map for `DiskStateProgressBar` severity, solved with introducing `enum EDiskStateSeverity`